sql_generators/glean_usage/templates/event_flow_monitoring_aggregates_v1.script.sql

```diff
@@ -168,10 +167,8 @@ CREATE TEMP TABLE
 
 MERGE
   `{{ project_id }}.{{ target_table }}` r
-USING
-  event_flows f
-ON
-  r.flow_id = f.flow_id
+  USING event_flows f
+  ON r.flow_id = f.flow_id
```

Comment:
@scholtzan this (mis)formatting of `MERGE` statements is unfortunate.  I think ideally `USING` in `MERGE` statements would be formatted like `JOIN`s are in `SELECT` statements (and IMO SQL should have gone with a `JOIN` keyword for this use case):
```suggestion
USING
  event_flows f
  ON r.flow_id = f.flow_id
```
However, implementing such a formatting variance for `USING` within `MERGE` statements seems like it'd be pretty tricky.  At the moment I'd prefer to merge this PR as is to get the readability improvements in the vast majority of queries and live with this minor `MERGE` formatting issue, but let me know if you disagree.

Reply:
Yeah, I can live with this